static/js/components/tiles/map_tile.tsx

```diff
@@ -749,3 +751,94 @@ function getExploreLink(props: MapTilePropType): {
     url: `${props.apiRoot || ""}${URL_PATH}#${hash}`,
   };
 }
+
+/**
+ * Get the HTML that can be used to embed this tile as a web component
+ * @param props this tile's props
+ * @returns HTML used to embed this tile as a web component
+ */
+export function getWebComponentSourceCode(mapChartData: MapChartData): string {
+  if (!mapChartData) {
+    return "";
+  }
+
+  const props = mapChartData.props;
+
+  // Get all places and types in the chart
+  const places = mapChartData.layerData
+    .map((layer) => layer.place.dcid)
+    .join(" ");
+  const enclosedPlaceTypes = mapChartData.layerData
+    .map((layer) => layer.enclosedPlaceType)
+    .join(" ");
+
+  // Get unique variables
+  const variables = _.uniq(
+    mapChartData.layerData.map((layer) => {
+      return layer.variable.statVar;
+    })
+  ).join(" ");
+
+  // Get list of variables that are per capita
+  const perCapitaVariables = mapChartData.layerData
+    .map((layer) => {
+      return layer.variable;
+    })
+    .filter((variable) => {
+      return variable?.denom == "Count_Person";
+    })
+    .map((variable) => {
+      return variable?.statVar;
+    })
+    .join(" ");
+
+  // Check if a specific date is being used for all variables
+  let date = "";
+  if (
+    mapChartData?.layerData.length > 0 &&
+    mapChartData.layerData.every((layer) => layer.variable.date)
+  ) {
+    date = mapChartData.layerData[0]?.variable?.date || "";
+  }
+
+  // Generate title with replacement strings filled in
+  const replacementStrings =
+    mapChartData && getReplacementStrings(props, mapChartData);
+  const header = getChartTitle(props.title, replacementStrings);
+
+  let sourceCode = `<script src="https://datacommons.org/datacommons.js"></script>
+<datacommons-map
+\theader="${header}"
+\tchildPlaceTypes="${enclosedPlaceTypes}"
+\tparentPlaces="${places}"
```

Comment:
Can you use childPlaceType & parentPlace (singular) if there's only a single parent place? Right now i dont think we actually use multiple parent places anywhere, and using multiple places isn't documented in the web component map api